addons/payment_mollie_official/models/res_partner.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+import logging
+import phonenumbers
+
+from odoo import api, models
+
+_logger = logging.getLogger(__name__)
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    def _get_mollie_address(self):

```

Comment:
delete this, parse the address in the acquirer model and simply never send the phone number
the phonenumber library is not always guaranteed to be installed